install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -27,8 +27,9 @@ data:
     enableTracing: true
     #
     # To disable the mixer completely (including metrics), comment out
-    # the following line
-    mixerAddress: {{ .Release.Name }}-mixer.{{ .Release.Namespace }}:15004
+    # the following lines
+    mixerCheckServer: {{ .Release.Name }}-mixer.{{ .Release.Namespace }}:15004
```

Comment:
Not sure what 'release.name' means - but if it's not istio-mixer many things are likely to break.